storage/filesystem/internal/dotgit/dotgit.go

```diff
@@ -51,9 +53,7 @@ func New(fs fs.FS, path string) (*DotGit, error) {
 // Symbolic references are resolved and included in the output.
 func (d *DotGit) Refs() ([]*core.Reference, error) {
 	var refs []*core.Reference
-	if err := d.addRefsFromPackedRefs(&refs); err != nil {
-		return nil, err
-	}
+	_ = d.addRefsFromPackedRefs(&refs)
```

Comment:
We cannot ignore errors, think of using a database to implement the underlying file system.